tests/ImageTests.php

```diff
@@ -447,7 +460,12 @@ protected function output($file)
     public function setUp()
     {
         $dir = $this->output('');
-        `rm -rf $dir`;
+        
+        if (PHP_OS === 'WINNT') {
+            `rd /s /q "$dir"`;
```

Comment:
So why not using Symfony FileSystem component?

http://symfony.com/doc/current/components/filesystem/introduction.html#remove